### PR TITLE
Automation for brief creation

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: scripts/automation/tests/brief/specialist.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const buyerLogin = require('../../useCases/login/buyer');
+const specialistBrief = require('../../useCases/brief/specialist');
+const buyerDashboard = require('../../useCases/dashboard/buyer');
+
+describe('should be able to create specialist brief', function () {
```
Comment (matt-sm) [inline scripts/automation/tests/brief/specialist.js]: general comment - try to ES6-ify this code eg 
- `function` should be `() =>`
- `exports.create` should be `export const create`

File: scripts/automation/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+To run this, first create an environment file with the following and fill in the [BLANKS]:
```
Comment (matt-sm) [inline scripts/automation/README.md]: perhaps replace what is currently in the `fixtures` folder ?

not sure `scripts` is the right place

File: scripts/automation/package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (matt-sm) [inline scripts/automation/package.json]: you should also commit the package lock file

File: package.json [hunk 4]
```diff
@@ -65,7 +65,7 @@
     "nodemon": "^1.12.1",
     "postcss-loader": "0.13.0",
     "prettier": "^1.5.3",
-    "puppeteer": "^1.2.0",
+    "puppeteer": "^1.5.0",
```
Comment (matt-sm) [inline package.json]: is this still needed in main package file?

Comment (malmas18) [reply package.json]: yep, for the web scraping

Comment (matt-sm) [reply package.json]: i mean - does it have to be in root package.json

File: scripts/automation/tests/brief/specialist.js (hunk 1 above)
Comment (malmas18) [reply scripts/automation/tests/brief/specialist.js]: fixed

File: scripts/automation/README.md (hunk 2 above)
Comment (malmas18) [reply scripts/automation/README.md]: fixed

File: scripts/automation/package.json (hunk 3 above)
Comment (malmas18) [reply scripts/automation/package.json]: added

File: package.json (hunk 4 above)
Comment (malmas18) [reply package.json]: removed from root package.json

File: fixtures/flows/brief/outcome.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
+export const create = async (params) => {
+    console.log('Starting to create outcome brief');
+    await selectLot('digital-outcome');
+    await createBrief();
+    await fillTitle(params.title);
+    await fillLocation(params.locations);
+    await fillDescriptinoOfWork();
```
Comment (tobiasbrown) [inline fixtures/flows/brief/outcome.js]: Typo in `Description` and in function definition.

File: fixtures/flows/brief/specialist.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
+export const create = async (params) => {
+    console.log(`Starting to create ${params.areaOfExpertise} brief`);
+    await selectLot('digital-professionals');
+    await createBrief();
+    await fillRole(params.title);
+    await selectLocation(params.locations);
+    await fillDescriptinoOfWork();
```
Comment (tobiasbrown) [inline fixtures/flows/brief/specialist.js]: Same typo here and in function definition.

File: fixtures/tests/bootstrap.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import puppeteer from 'puppeteer';
+import { expect } from 'chai';
+import * as utils from '../flows/utils';
+
+
+// puppeteer options
+const opts = {
+    headless: false,
+    //slowMo: 100,
+    //timeout: 10000
```
Comment (tobiasbrown) [inline fixtures/tests/bootstrap.js]: Is this a WIP or something that can be removed / turned into comments?

File: fixtures/flows/brief/outcome.js (hunk 5 above)
Comment (malmas18) [reply fixtures/flows/brief/outcome.js]: fixed

File: fixtures/flows/brief/specialist.js (hunk 6 above)
Comment (malmas18) [reply fixtures/flows/brief/specialist.js]: fixed

File: fixtures/tests/bootstrap.js (hunk 7 above)
Comment (malmas18) [reply fixtures/tests/bootstrap.js]: fixed

**Pull request merged**